crates/red_knot_python_semantic/resources/mdtest/type_of/typing_dot_Type.md

```diff
@@ -0,0 +1,32 @@
+# `typing.Type`
+
+## Annotation
+
+`typing.Type` can be used interchangeably with `type`:
+
+```py
+from typing import Type
+
+class A: ...
+
+def _(c: Type, d: Type[A], e: Type[A]):
```

Comment:
Why does the `e` argument exist here?

Reply:
Sorry, I think I screwed that up when simplifying the mdtests a little :-)